File: java/terminal/terminal-core/src/main/java/com/github/fernthedev/terminal/core/packets/MessagePacket.java
package com.github.fernthedev.terminal.core.packets;

import com.github.fernthedev.lightchat.core.packets.Packet;
import com.github.fernthedev.lightchat.core.packets.PacketInfo;

@PacketInfo(name = "MESSAGE_PACKET")
public class MessagePacket extends Packet {

    public MessagePacket(String message) {
        this.message = message;
    }

    private String message;

    public String getMessage() {
        return message;
    }

    @Override
    public String toString() {
        return "Packet Message: " + message;
    }
}

File: java/terminal/terminal-core/src/main/java/com/github/fernthedev/terminal/core/packets/MessagePacket.kt
package com.github.fernthedev.terminal.core.packets

import com.github.fernthedev.lightchat.core.packets.PacketJSON
import com.github.fernthedev.lightchat.core.packets.PacketInfo

@PacketInfo(name = "MESSAGE_PACKET")
class MessagePacket(@JvmField val message: String) : PacketJSON() {

    override fun toString(): String {
        return "Packet Message: $message"
    }
}